ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -86,6 +123,8 @@ void AddMeshNode(string name, VMesh mesh, Skeleton skeleton)
                 AddMeshNode(meshes[i].Name, meshes[i].Mesh, model.GetSkeleton(i));
             }
 
+
+
```

Comment:
there's a few pointless lines that got added in this file

Reply:
fixed extra lines in this file